cmd/livepeer/livepeer.go

```diff
@@ -98,6 +100,7 @@ func main() {
 	monitor := flag.Bool("monitor", true, "Set to true to send performance metrics")
 	monhost := flag.String("monitorhost", "http://viz.livepeer.org:8081/metrics", "host name for the metrics data collector")
 	ipfsPath := flag.String("ipfsPath", fmt.Sprintf("%v/.ipfs", usr.HomeDir), "IPFS path")
+	noLogFiles := flag.Bool("noLogFiles", false, "Set to true if log files should not be generated")
```

Comment:
Does this affect all logs or just IPFS logs for now?

Reply:
Just IPFS logs for now

Reply:
Maybe rename the flag to "noIPFSLogFiles" for now?